community-operators/gitops-operator/0.0.1/pipelines.openshift.io_gitopsservices_crd.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
@chetan-rns, please use apiextensions.k8s.io/v1beta1 if possible

Reply:
I've changed the version. Can you please take a look?

Reply:
Hi @J0zi , I don't have access to retest the changes. Retriggering Travis may fix this error